bot/cogs/image_dots_cog.py

```diff
@@ -0,0 +1,193 @@
+        new_img = []
+        try:
+            # meant when the person wants to exclude the embed in a regular message
+            # breaks the bot otherwise
+            filename = filename.replace('<', '').replace('>', '')
+            with Image.open(requests.get(filename, stream=True).raw) as img:
+                # default asciiWidth I found online. Aparently over 500 gets laggy
+                new_img = self.parse_image(img, width, threshold, inverted)
+                await ctx.send('\n'.join(new_img))
+        except UnidentifiedImageError as e:
+            embed = discord.Embed(title=f'ERROR: unable to open message link', color=Colors.Error)
+            embed.add_field(name='Exception:', value=filename + ' link does not exist, or is broken!')
+            await ctx.send(embed=embed)
+
+    @ext.group(invoke_without_command = True)
+    @ext.long_help('Takes any image, and returns the brailled image.\nCan specify mobile or pc width characteristics.\nDefault width is pc size.\
+                    Choose a width of \'pc\', \'mobile\', or leave blank.\n \
+                    Threshold determines which pixels are white, and which are blank. Choose a value [0-255]\n\
+                    The final argument asks, do you want to invert the image or not?\n \
+                    When attachment specifier is used, you can upload an image directly. All other arguments \n \
+                    are the same except you no longer need a url for the first argument')
+    @ext.short_help('Turn an image to a braille image.\nDefault width is pc size.\nDefault threshold is 150.\n\
+                    To specify threshold you must include all required arguments.\n \
+                    The same goes for all arguments')
+    @ext.example(('todots https://my-cool-image.com/stuff.jpg [mobile|pc] [threshold = 0-255] [inverted = 0/1]',
+    'todots https://my-cool-image.com/stuff.jpg [mobile|pc] [threshold = 0-255]',
+    'todots https://my-cool-image.com/stuff.jpg [mobile|pc]', 
+    'todots https://my-cool-image.com/stuff.jpg'))
+    async def todots(self, ctx, image, device = None, threshold = 150, inverted = 0) -> None:
+        return await self.todots_helper(ctx,image,device,threshold,inverted)
+
+    @todots.command()
+    @ext.long_help('Takes any image, and returns the brailled image.\nCan specify mobile or pc width characteristics.\nDefault width is pc size.\
+                    Choose a width of \'pc\', \'mobile\', or leave blank.\n \
+                    Threshold determines which pixels are white, and which are blank. Choose a value [0-255]\n\
+                    The final argument asks, do you want to invert the image or not?\n \
+                    When attachment specifier is used, you can upload an image directly. Must attach an image.')
+    @ext.short_help('Turn an attached image to a braille image.\nDefault width is pc size.\nDefault threshold is 150.\n\
```

Comment:
we dont want to do line breaks manually here because it wont look right on all screens. instead we use python implicit screen concatencation